lib/atomic-management.js

```diff
@@ -294,10 +289,14 @@ export default {
    * @param {Array<String>} packageNames The names of packages that are not installed currently
    */
   askInstallPackages(packageNames) {
+    if (!atom.config.get('atomic-management.enforceConfiguredPackages')){
```

Comment:
Can you add a menu option to toggle enforce configured packages and then a little thing in the readme about what it means to "enforce configured packages", saying that it defaults to false